### PR TITLE
Make the images the same as the template

### PR DESCRIPTION
**Pull request by x5hwuk7nc6jsudcc**

### CODE REVIEW

File: _includes/home_testimonials.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h2 class="font-weight-bold">
                 <!-- Icon -->
                 <div class="img-fluid mb-5 w-50 svg-shim mx-auto" style="color: #FF5A5F;">
                   {% if company.logo_name contains ".svg" %}
-                  <object data="/assets/img/clients/{{ company.logo_name }}" type="image/svg+xml"
+                  <img src="/assets/img/clients/{{ company.logo_name }}" type="image/svg+xml"
```
Comment (davidgatti) [inline _includes/home_testimonials.html:38]: @x5hwuk7nc6jsudcc how is this related to the image in the articles?

**Review by davidgatti — CHANGES_REQUESTED** [review]
One thing to fix or give more context since I'm not sure I understand the change for this PR.

File: _includes/home_testimonials.html (hunk 1 above)
Comment (x5hwuk7nc6jsudcc) [reply _includes/home_testimonials.html]: Apologies. I didn't create a clean branch properly. The above code is actually already merged into `master` from https://github.com/0x4447-office/0x4447_webpage_consulting/pull/301 so I have re-sync'd with `master` and you'll see in the 'files changed' above that this PR will no longer make the above code change.

**Review by davidgatti — APPROVED** [review]
